docs/docs.go

```diff
@@ -0,0 +1,114 @@
+// Package docs Code generated by swaggo/swag. DO NOT EDIT
```

Comment:
I think we can change the directory name `docs` to `api`. can we set the directory to create swagger doc?
https://github.com/golang-standards/project-layout/blob/master/README.md#api

Reply:
We can config another dir for swagger docs. I updated!